src/js/GaCesium.js

```diff
@@ -178,3 +181,93 @@ var GaCesium = function(map, gaPermalink, gaLayers, gaGlobalOptions, $q) {
 
 };
 
+var SSECorrector = function() {
+  var params = {};
+  location.search.substr(1).split('&').reduce(function(previous, current) {
+    var splitted = current.split('=');
+    if (splitted.length === 2) {
+      params[splitted[0]] = splitted[1];
+    } else {
+      params[splitted[0]] = true;
+    }
+  });
+
+  this.mindist = parseInt(params['mindist'] || '5000', 10);
+  this.maxdist = parseInt(params['maxdist'] || '10000', 10);
+  this.mincamfactor = parseFloat(params['mincamfactor'] || '0.9');
+  this.maxcamfactor = parseFloat(params['maxcamfactor'] || '1.2');
+  // Max height to apply optmization
+  this.maxheight = parseInt(params['maxheight'] || '0', 10);
+  this.allowtilelevels = parseInt(params['allowtilelevels'] || '0', 10);
+  this.pickglobe = !params['nopickglobe'];
+  this.pickposition = parseFloat(params['pickposition'] || '0.6666');
+  this.shouldCut = !params['nocut'];
+  this.noheight = !!params['noheight'];
+  this.maxerrorfactor = parseFloat(params['maxerrorfactor'] || '0.25');
+  this.cameraHeight = undefined;
+};
+
+/**
+ * Called before the Quadtree will update its tiles.
+ * @param {Cesium.FrameState} frameState
+ */
+SSECorrector.prototype.newFrameState = function(frameState) {
+    this.cameraHeight = frameState.camera.positionCartographic.height;
+
+    if (this.pickglobe) {
+      var scene = frameState.camera._scene;
+      var canvas = scene.canvas;
+      var pixelHeight = this.pickposition * canvas.clientHeight;
+      var pixel = new Cesium.Cartesian2(canvas.clientWidth / 2, pixelHeight);
+      this.cameraHeight = undefined;
```

Comment:
Without this code, there is a big difference between the camera height (vertical) and the distance to the center of the screen (oblique). With an oblique view, all tiles could be blured.